src/gpgmm/d3d12/ResourceHeapAllocatorD3D12.h

```diff
@@ -23,6 +23,18 @@ namespace gpgmm::d3d12 {
 
     class ResidencyManager;
 
+    class TryAllocateMemoryCallbackContext {
```

Comment:
unit: rename to `CreateResourceHeapCallbackContext` to be consistent?

Reply:
Done.